README.md

```diff
@@ -20,24 +19,32 @@ You'll need the following to run the template. [Homebrew](https://brew.sh/) for
 ## Getting started
 
 1. Open a shell, clone the repository, and navigate to its root.
-    ```
+    ```bash
     git clone https://github.com/gslab-econ/template.git
     cd template
     ```
-2. You're ready to go. We'll prompt you to enter any necessary information and store it in `config_user.yaml` as your scripts run. 
-    * To build everything that has been modified or with dependencies in the repository that have been modified.
-        ```
-        scons
+2. Unzip `scons.zip` to `scons`.
+    ```bash
+    unzip scons.zip -d scons
```

Comment:
I think Windows does not support `unzip` command.

Reply:
You're right, and I was torn on whether to include code for this step. I like the code because it shows exactly where we unzip to. Maybe a comment that says not windows? I'd also get rid of the code entirely if you think that's smarter.

Reply:
I think it's good to have the code. Just make a note perhaps.